src/cclib/parser/gaussianparser.py

```diff
@@ -1523,12 +1523,24 @@ def extract(self, inputfile, line):
         #Sum of electronic and thermal Energies=              -563.636699
         #Sum of electronic and thermal Enthalpies=            -563.635755
         #Sum of electronic and thermal Free Energies=         -563.689037
-        if "Sum of electronic and thermal Enthalpies" in line:
-            self.set_attribute('enthalpy', float(line.split()[6]))
-        if "Sum of electronic and thermal Free Energies=" in line:
-            self.set_attribute('freenergy', float(line.split()[7]))
         if line[1:12] == "Temperature":
             self.set_attribute('temperature', float(line.split()[1]))
+        if "Sum of electronic and zero-point Energies=" in line:
```

Comment:
Yes, it should be subtracted out. Though, why not use the line with `Zero-point correction=`?